DOC.md

```diff
@@ -42,6 +42,7 @@ local fmt = require("luasnip.extras.fmt").fmt
 local fmta = require("luasnip.extras.fmt").fmta
 local conds = require("luasnip.extras.expand_conditions")
 local postfix = require("luasnip.extras.postfix").postfix
+local sl = require("luasnip.extras.snippet_list")
```

Comment:
Oh, this list is mainly meant for shorthands used when defining snippets, and I don't think `sl` is useful there :D
Maybe introduce this shorthand in the `snippet_list`-section itself?

Reply:
Moved the shorthand to the snippet list section!